Games/types/Mafia/roles/cards/WinFourPolarisedDeaths.js

```diff
@@ -12,7 +12,7 @@ module.exports = class WinFourPolarisedDeaths extends Card {
         if (
           this.player.alive &&
           !winners.groups[this.name] &&
-          (this.polarisedKills >= 4 || aliveCount == 2)
+          (this.polarisedKills >= 4 || counts["Polar Bear"] >= aliveCount / 2 && aliveCount > 0)
```

Comment:
counts is only for alignments not roles

Reply:
what to use in place? this.player.role.name === "Polar Bear" would work?